kotlin/android/app/src/main/java/dev/firezone/android/core/data/Repository.kt

```diff
@@ -14,6 +15,40 @@ import kotlinx.coroutines.flow.flowOn
 import java.security.MessageDigest
 import javax.inject.Inject
 
+public const val ON_SYMBOL: String = "<->"
+public const val OFF_SYMBOL: String = " — "
```

Comment:
Should these be defined on the companion object at the bottom?

Also I think `public` might be redundant?

Reply:
> Should these be defined on the companion object at the bottom?

Is there any benefit to that? Also, shouldn't in that case be a companion to `ResourceState`?